js-exercises/sum-all/sumAll.js

```diff
@@ -0,0 +1,15 @@
+function sumAll(numbers) {
+  let num1 = numbers[0];
+  let num2 = numbers[1];
+  let totalTerms = 0;
+  if (num1 > num2) {
+    totalTerms = num1 - num2
+  } else {
+    totalTerms = num2 - num1
+  }
```

Comment:
This could be replaced with a single line

`const totalTerms = Math.abs(num1 - num2);`